dev/text-area.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <script type="module">
+      import '@vaadin/text-area';
+    </script>
+  </head>
+
+  <body>
+    <vaadin-text-area label="Required" required></vaadin-text-area>
+
+    <div>Is valid? <span id="validity"></span></div>
+
+    <script>
+      const area = document.querySelector('vaadin-text-area');
+      const validity = document.querySelector('#validity');
+
+      area.addEventListener('change', (e) => {
```

Comment:
Same issue as `vaadin-email-field` and `vaadin-password-field`
```suggestion
      area.addEventListener('focusout', (e) => {
```

Reply:
Removed this listener.